src/main/java/at/ac/tuwien/kr/alpha/common/predicates/Evaluable.java

```diff
@@ -0,0 +1,11 @@
+package at.ac.tuwien.kr.alpha.common.predicates;
+
+import at.ac.tuwien.kr.alpha.common.terms.Term;
+import at.ac.tuwien.kr.alpha.grounder.Substitution;
+
+import java.util.List;
+
+@FunctionalInterface
+public interface Evaluable {
```

Comment:
From what I understand this interface should cover atoms with a fixed interpretation similar as builtin atoms. Some other name might be more fitting then (maybe FixedInterpretationEvaluable?)

Reply:
Yes, they have a fixed interpretation. Maybe only `FixedEvaluable` suffices? `FixedInterpretationEvaluable` is quite long :)